source/standards/logging.html.md.erb

```diff
@@ -26,6 +26,9 @@ they are available and queryable.  This should not be considered a
 durable store of logs; it is *not* suitable for long-term archival of
 logs.
 
+Guides on how to use the logit service can be found in
+the [Reliability Engineering team documentation]().
```

Comment:
should this be `[Reliability Engineering team documentation][]` (square brackets not round parens)?